src/main/resources/lib/credentials/select.jelly

```diff
@@ -87,13 +87,24 @@
             <div class="bd">
             <ul class="first-of-type">
             <j:forEach var="storeItem" items="${storeItems}">
-              <li class="yuimenuitem" data-url="${request.contextPath}/${storeItem.url}/dialog"
-                  disabled="${storeItem.enabled?null:'disabled'}">
-                <span class="yuimenuitemlabel" tooltip="${storeItem.description}">
-                <l:icon class="${storeItem.iconClassName} icon-sm"/>
-                ${storeItem.displayName}
-                </span>
-              </li>
+              <j:choose>
+                <j:when test="${storeItem.enabled}">
+                  <li class="yuimenuitem" data-url="${request.contextPath}/${storeItem.url}/dialog">
+                    <span class="yuimenuitemlabel" tooltip="${storeItem.description}">
+                      <l:icon class="${storeItem.iconClassName} icon-sm"/>
+                      ${storeItem.displayName}
+                    </span>
+                  </li>
+                </j:when>
+                <j:otherwise>
+                  <li class="yuimenuitem" disabled="disabled">
+                    <span class="yuimenuitemlabel" tooltip="${storeItem.description}" disabled="disabled">
```

Comment:
Would make for a smaller diff and simpler code to introduce a variable for `storeItem.enabled ? null : 'disabled'` and use it on both the `li` and the `span`.